# Total planning attempts
planning_attempts: 10

# Planning group and link names
arm_group_name: "panda_arm"
eef_name: "hand"
hand_group_name: "hand"
hand_frame: "panda_link8"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"

# Gripper grasp frame transform [x,y,z,r,p,y]
ik_frame: [0, 0, 0.17, 1.571, 0.785, 1.571]

